scripts/flash/index.js

```diff
@@ -19,35 +19,92 @@ var marketInfo = require("./market-info");
 var showInitialReporter = require("./show-initial-reporter");
 var fork = require("./fork");
 var approval = require("./approve-account");
+var listMarketOrders = require("./list-market-orders");
+var fillMarketOrders = require("./fill-market-orders");
 
-var commands = ["get-balance", "list-markets", "designate-report", "initial-report", "dispute-contribute", "finalize-market", "push-time", "market-info", "show-initial-reporter", "fork", "approval"];
 var NETWORKS = ["aura", "clique", "environment", "rinkeby", "ropsten"];
-var methods = [getBalance, listMarkets, designatedReport, initialReport, disputeContribute, finalizeMarket, pushTime, marketInfo, showInitialReporter, fork, approval];
+var methods = [
```

Comment:
Much easier to read! I think it could be even more improved if the key was the command and the value was the method, but this is def an improvement